com.valvesoftware.Steam.json

```diff
@@ -95,7 +95,45 @@
                 "url": "https://mirrors.edge.kernel.org/pub/software/utils/pciutils/pciutils-3.6.2.tar.gz",
                 "sha256": "d5f9254f27bbda8243b345633e980144e6bd2af9c786bb8a152b904530aef599"
             }]
-        },        
+        },
+	{
+	    "name": "systemd",
```

Comment:
Are you sure all of this is required? Did you see https://github.com/flathub/com.valvesoftware.Steam/commit/4c74ac232da3406cf3e0a8129d126773c38b8d8a ?